tests/test_query.py

```diff
@@ -29,14 +58,14 @@ def mock_successful_get(test_data):
     """Mock get function that returns non-empty query results."""
 
     async def mockreturn(
-        age_min,
-        age_max,
+        min_age,
+        max_age,
         sex,
         diagnosis,
         is_control,
         min_num_sessions,
-        image_modal,
         assessment,
```

Comment:
Why is this now moved up to here? No issue with re-ordering arguments. But there should be a reason, or at least we should agree on one way so we don't reshuffle this every couple of PRs.

Reply:
The `assessment` is moved above `image_modal` and below `min_num_sessions` to follow the filtering order inside `create_query`. Since we don't have inline comments inside different modules I thought having consistent order in which parameters/arguments appear across different modules can help with readability especially for a module with a lot of code e.g., `test_query`

Reply:
Ah OK, makes sense to me. Thanks for the explanation